scripts/postbuild.sh

```diff
@@ -5,5 +5,5 @@ set -e
 # Generate sitemap
 pnpm exec next-sitemap
 
-# Update blog search data
-curl -o public/feed.json  https://authzed.com/feed/json
+# Generate search
+pagefind --site .next/server/app --output-path public/_pagefind --verbose
```

Comment:
This is the command that generates the search index that's used by the search bar.